### ISSUE TITLE
Level: "Tetris"

### ISSUE THREAD

**Issue by colonelfazackerley** [andybalaam/rabbit-escape#158]
This makes 20 levels for the arcade set.

I image@tttppp will be able to get out more than 5. Not all the tokens are necessary, but the level is long and fiddly, so I wanted to make it a little more forgiving.

```
:name=Tetris
:bridge=20
:bash=4
:block=4
:explode=4
:num_rabbits=10
:num_to_save=5
:rabbit_delay=30,30,30,10
    O#   
    ##   
    # ## 
      ## 
 ###     
 #    #  
     ##  
 Q## #   
  ##   # 
       # 
  ###  # 
   #   # 
 #    #  
 #  ###  
 ##      
```

**Comment by colonelfazackerley** [comment]
merged to master f96dd5696bc0d3c0f44b4856ba75e3a7d2bc201a

**Comment by tttppp** [comment]
Wow, what a level! I think I should be able to get six out, but this crashes on my phone when trying to explode blockers (same problem as K2). I really like the level, possibly my favourite of yours. I've only found one solution (with quite a few minor variations), and this makes me think you must have spent quite a while designing it. Especially given the constraint that it's entirely tetrominoes.

**Comment by colonelfazackerley** [comment]
I didn't change the blocks much. It took quite a while to play through, especially in text mode.

**Comment by andybalaam** [comment]
Cool level, surprisingly hard.

**Comment by andybalaam** [comment]
@tttppp I am assuming the crashes are out of memory. I get that easily on my phone if I zoom in. After getting arcade out I need to focus on stability and that problem is top of the list.

**Comment by andybalaam** [comment]
Hint 2 has an extra apostrophe and hint 3 has a missing space.

**Comment by tttppp** [comment]
I got seven out by optimising the timing for the only method I've found. Can't beat seven at the moment.

**Comment by tttppp** [comment]
@andybalaam Yes, out of memory. It pretty much always strikes on this level when exploding blockers at the end.

**Comment by andybalaam** [comment]
Hints fixed in 520621268bc72e5f717eb62aaf3c0d5e87138721

**Comment by andybalaam** [comment]
Saving 7 a possibility for #87 Deja vu?

**Comment by andybalaam** [comment]
I just saved 8 with:

```
solution.1=bridge;(3,13);;block&bridge&bash&bridge;(5,12);8;(7,11);2;(6,10);6;bash;;(6,11);bridge;(6,11);10;(3,9);block;11;(0,7);;bridge;2;(1,8);bash;2;(2,8)&bridge;3;(4,7);8;(3,7);9;(6,5)&bash;2;(5,5)&bridge;;(5,5);4;(2,3);block;6;(0,2);bridge;3;(1,3);74;explode;2;(0,2);(0,7);
```

Any objections to me making it 8 for #87 Deja Vu?

**Comment by tttppp** [comment]
Save eight sounds like a good challenge! I tried this recently and only got up to seven again.

**Comment by tttppp** [comment]
Save eight was surprisingly easy once I knew it was possible! I tried for quite a long time before you said you'd managed it, but once I knew it was possible it only took 5-10 minutes or so!

Great level for Deja Vu.

**Comment by andybalaam** [comment]
Yeah I was amazed how easy it was. I remember this level being really hard with its original target. Anyway, it's in snapshot 0.9.1.3 ready to be released in 0.10.
